### ISSUE TITLE
Hoverboard.axf not found

### ISSUE THREAD

**Issue by bugiworks** [flo199213/Hoverboard-Firmware-Hack-Gen2#3]
I appreciate your work, after tried one motherboard hoverboard, i want to try this split one also. 

when i try to build with keil, i get the error
.\Objects\Hoverboard.axf: error: L6031U: Could not open scatter description file .\Objects\Hoverboard.sct: No such file or directory
Not enough information to list image symbols.
Not enough information to list load addresses in the image map.
Finished: 2 information, 0 warning, 0 error and 1 fatal error messages.
".\Objects\Hoverboard.axf" - 1 Error(s), 0 Warning(s).
Target not created.

any solution? or any direct hex files do you have?
By the way, the board im using has GD32F103 not F130, i have downloaded the pack for 103 but im not sure if it works same.

**Comment by pieterjanbuntinx** [comment]
Have you found any solution for this error? I'm experiencing the same

**Comment by kevin0932** [comment]
Hi @bugiworks @pieterjanbuntinx , I met the same error. Did you manage to solve it? Thanks for sharing.

**Comment by pieterjanbuntinx** [comment]
Hi @kevin0932,
No, I did not. But I found [a fork](https://github.com/btsimonh/Hoverboard-Firmware-Hack-Gen2) of this repo that compiles. I have not managed to get it uploaded to my hoverboard controller, I don't have a STLink but I'm using a STM32 Discovery board, so that's probably the problem.

**Comment by bugiworks** [comment]
Unfortunately i couldnt find any solution, I will try that fork then i will update the comment.

**Comment by pieterjanbuntinx** [comment]
It now works for me. To solve this error I had to enable this: 
![image](https://user-images.githubusercontent.com/34648386/53434955-f6183e00-39f7-11e9-8ce5-59f4a3d48d82.png)

**Comment by renmalek7** [comment]
Hi. Were you able to flash this firmware?

**Comment by pieterjanbuntinx** [comment]
Yes, I was able to flash it. It works great. What are you stuck on?

**Comment by renmalek7** [comment]
I have a different board, it's also a GD32F130. Before flashing the firmware, How can I tell if the Pins are the same?
![2](https://user-images.githubusercontent.com/62514161/78782016-984b7900-79a1-11ea-8a05-c957ce13bea8.jpg)

**Comment by pieterjanbuntinx** [comment]
You could perhaps measure continuity using a multimeter from the pins of the MCU to the peripherals. If that corresponds to the pinout in the code it should work. That will be quite some work though, so if you think that the layout of the PCB looks about the same as the one of this repo, you could perhaps just try it out (at your own risk). Replacement boards are not that expensive from AliExpress

**Comment by renmalek7** [comment]
ok, I will do that and post the results. Do you know if there is a way to pull the original firmware from the board? Would it be posible to write it back in the board if I fail and its not burned l?

**Comment by pieterjanbuntinx** [comment]
No I don't think so. I thought it was locked or something. It has something to do with the options bytes

**Comment by renmalek7** [comment]
ok, thank you very much for you time : )

**Comment by Harsh2210** [comment]
> I have a different board, it's also a GD32F130. Before flashing the firmware, How can I tell if the Pins are the same?
> ![2](https://user-images.githubusercontent.com/62514161/78782016-984b7900-79a1-11ea-8a05-c957ce13bea8.jpg)

Were you able to flash the firmware? I am using the same motherboard. I wanted to know whether the pinouts are same for this one.

**Comment by yodaco** [comment]
> > I have a different board, it's also a GD32F130. Before flashing the firmware, How can I tell if the Pins are the same?
> > ![2](https://user-images.githubusercontent.com/62514161/78782016-984b7900-79a1-11ea-8a05-c957ce13bea8.jpg)
> 
> Were you able to flash the firmware? I am using the same motherboard. I wanted to know whether the pinouts are same for this one.

Did it work?!?
